src/client/components/App.jsx

```diff
@@ -17,13 +19,16 @@ const App = () => {
     >
       <Routes>
         <Route path="/teamproject-react-node-soyummy" element={<StartPage />} />
-        {/* <Route path="/categories/:categoryName" element={<CategoriesPage />} />
+        <Route path="/" element={<SharedLayout />}>
```

Comment:
I quess it only implement to check how it look.
Because if you want to create `SharedLayout` like you do, you have to serve this component as a `div` for rest routes/components passing `main` section as a `children` props.

f.ex.
`const SharedLayout = ({ children }) => { ... return ( <SharedLayoutElements /> { children } ) }`

```
<SharedLayout>
  <Component />
<SharedLayout />
```

Reply:
Take a look:
<Routes>
        <Route path="/teamproject-react-node-soyummy" element={<StartPage />} />
        <Route path="/" element={<SharedLayout />}>
          <Route index element={<Home />} />
          <Route path="/categories/:categoryName" element={<Categories />} />
          <Route path="/add" element={<AddRecipes />} />
          <Route path="/my" element={<MyRecipes />} />
          <Route path="/favorite" element={<Favorites />} />
          <Route path="/shopping-list" element={<ShoppingList />} />
          <Route path="/search" element={<Search />} />
          <Route path="*" element={<Page404 />} /> 
        </Route>
      </Routes>
All the routes are nested in Route component containing SharedLayout element, so their elements are rendered within SharedLayout as Outlet component. It really works, I checked it a few times :)

Reply:
Hmm it is not the same code I have just pasted...

Reply:
![Uploading Zrzut ekranu 2024-02-19 o 22.49.00.png…]()

Reply:
<img width="1440" alt="Zrzut ekranu 2024-02-19 o 22 49 00" src="https://github.com/KaczorowskaKarolina/teamproject-react-node-soyummy/assets/126253505/2f1b3455-e3c3-4ff1-9647-5d0a81664239">

Reply:
That's why I suggest you to change it 🤞

Reply:
❓

Reply:
Still working on it? ❓